gammapy/irf/tests/test_background.py

```diff
@@ -49,3 +53,64 @@ def test_background_3d_write(bkg_3d):
     hdu = table_to_fits_table(bkg_3d.to_table())
     assert_equal(hdu.data['DETX_LO'][0], bkg_3d.data.axis('detx').lo.value)
     assert hdu.header['TUNIT1'] == bkg_3d.data.axis('detx').lo.unit
+
+
+def make_test_array():
+    # Create a dummy `Background2D`
```

Comment:
I think it would be better to have an even simpler test object. (even less code, easier to think about in the test methods below)

How about this?
```
energy = [1, 3, 5, 10] * u.TeV
offset = [0, 1, 5] * u.deg
data = energy.value * offset.value * u.Unit('s-1 MeV-1 sr-1')
return Background2D(
    energy_lo=energy[:-1], energy_hi=energy[1:],
    offset_lo=offset[:-1], offset_hi=offset[1:],
    data=data,
)
```
This is small, i.e. fast. And it's less code. And it's simpler to see what the expected value is in evaluate calls, if you choose a few points carefully (e.g. at a node and in the middle between two nodes, and throw in a third case that leads to extrapolation (e.g. offset=0) to establish behaviour).

I think we should do all our IRF tests like this: have super-simple test cases and fill them with data values that is easy and generic to write when making the test case, and also to know what to expect in evaluate.